docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -344,6 +344,24 @@ paths:
             Alias of `uri`.
           required: false
           type: string
+        - name: wildcard_uri
+          in: query
+          description: |
+            Limit the results to annotations matching the wildcard URI. 
+            URI can be a URL (a web page address) or a URN representing another 
+            kind of resource such as DOI (Digital Object Identifier) or a 
+            PDF fingerprint.
+
+            `*` will match any character sequence (including an empty one), 
+            and a `?` will match any single character. `*`'s and `?`'s are not
+            permitted within the domain of the URI.
```

Comment:
Strictly speaking they are not permitted in the scheme either. Perhaps easier to express this as "wildcards are only permitted within the path and query parts of the URI"?

Reply:
I've updated this based on the changes to the other PR so now we do allow `?`s and `*`s in certain parts of the netloc. Hopefully this description makes sense and isn't complicated to follow. I'm happy to change it again if need be.